QuickFIXn/DataDictionary/DataDictionary.cs

```diff
@@ -336,18 +338,20 @@ public void CheckValidTagNumber(int tag)
 		/// <param name="field"></param>
 		public void CheckValue(Fields.IField field)
 		{
```

Comment:
Please straighten out these indents.  Change all tabs to spaces.

Reply:
Oops, ignore that comment.  I'm mistaken.

Reply:
This whole file has screwy indents.  I will fix it later on my own branch.  Don't worry about it.